cli/internal/telemetry/telemetry.go

```diff
@@ -0,0 +1,102 @@
+package telemetry
+
+import (
+	"fmt"
+	"os"
+	"sync"
+
+	"github.com/google/uuid"
+	"github.com/rudderlabs/analytics-go/v4"
+	"github.com/rudderlabs/rudder-iac/cli/internal/config"
+)
+
+var (
+	once              sync.Once
+	v                 string
+	telemetryDisabled bool
+)
+
+func getEnvWithFallback(key, fallback string) string {
+	if value := os.Getenv(key); value != "" {
+		return value
+	}
+	return fallback
+}
+
+func Initialise(version string) {
+	once.Do(func() {
+		conf := config.GetConfig()
+
+		v = version
+		if conf.Telemetry.Disabled {
+			telemetryDisabled = true
+			return
+		}
+
+		disabled := getEnvWithFallback("RUDDER_CLI_TELEMETRY_DISABLED", "")
```

Comment:
New comment: I think we should prefix the env with `RUDDERSTACK` instead of `RUDDER_CLI` so that all envs have a common prefix. `RUDDERSTACK_CLI_TELEMETRY_DISABLED` works as well.